Project-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js

```diff
@@ -1,2 +1,12 @@
 import React from 'react';
 import './Header.css';
+
+export default function HeaderTitle(props) {
+    return (
+        <div className="header-title">
+            <h1>{props.title || "Lambda School"}</h1>
```

Comment:
Good use of conditional rendering here. this pattern is advanced. WHO ARE YOU ASHLEY!?